connector_onpay/models/hr_expense.py

```diff
@@ -0,0 +1,13 @@
+# Copyright (C) 2020 Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields
+
+
+class HrExpense(models.Model):
+    _inherit = 'hr.expense'
+
+    expense_onpay_id = fields.Many2one(
```

Comment:
Following on my previous comment, here `onpay_id` is an actual many2one, not a char code...